flintrock/flintrock.py

```diff
@@ -319,6 +319,17 @@ def cli(cli_context, config, provider, debug):
               multiple=True,
               help="Additional tags (e.g. 'Key,Value') to assign to the instances. "
                    "You can specify this option multiple times.")
+@click.option('--ec2-authorize-access-from',
+              callback=ec2.cli_validate_ec2_authorize_access,
+              type=str,
+              default=None,
+              multiple=True,
+              help="Authorize cluster access from a private source"
+                   "(To be used in VPC context, behind a nat gateway for instance)"
+                   "which can be either a plain IP,"
+                   "an IP network in CIDR notation, or an EC2 SecurityGroup."
+                   "For security concerns, this options disable client configuration "
+                   "using automatic detection of client public IP")
```

Comment:
I forget if we already discussed this, but does the `add-slaves` command automatically respect this setting? I assume yes, since this is done at the level of the cluster security group, but just confirming.

Reply:
Slaves get their security groups from cluster master ones, so it's not needed:

https://github.com/nchammas/flintrock/blob/d3f769e40d1cce742996412b6a2dc2956870fc93/flintrock/ec2.py#L258-L260